src/common-ui/containers/IndexDropdown.tsx

```diff
@@ -188,14 +194,19 @@ class IndexDropdownContainer extends Component<Props, State> {
     private addTag = async () => {
         const newTag = this.getSearchVal()
 
-        if (this.allowIndexUpdate) {
-            this.addTagRPC({
-                url: this.props.url,
-                tag: newTag,
-                tabId: this.props.tabId,
-            }).catch(console.error)
+        if (this.props.allTabs) {
```

Comment:
Reason not prepending `this.` as I mentioned earlier was because of `RemoteError: Got no response when trying to call 'fetchPageTags'. Did you enable RPC in the background script?`
I get this error only when I use `this.addTagsToOpenTabsRPC`. As I couldn't figure out the reason I imported `addTagsToOpenTabs` it for time being which works without error.

Reply:
That error is most likely because `fetchPageTags` is made remotely callable from the BG script, but then you're trying to do a `remoteFunction` call on it from the same BG script (in `search/tags` module). 

It should work fine now that you've imported it from a user POV, but it means you've duplicated the BG script in this UI script because of that `import` (the point 1 in my big comment). So yeah, I'd advise you to remove the `import`s here and replace them with the remote calls (even if they don't work just yet; next step is getting the BG code working)